app/views/admin/enterprise_relationships/_enterprise_relationship.html.haml

```diff
@@ -6,6 +6,7 @@
   %td
     %ul
       %li{"ng-repeat" => "permission in enterprise_relationship.permissions"}
-        to {{ EnterpriseRelationships.permission_presentation(permission.name) }}
+        = t 'admin_enterprise_relationships_to'
+        {{ EnterpriseRelationships.permission_presentation(permission) }}
```

Comment:
is there a reason why we change from `permission_presentation(permission.name)` to `permission_presentation(permission)`?

Reply:
My mistake, I'll make this change.